### ISSUE TITLE
Contrasts in default and Sepia themes fail accessibility test

### ISSUE THREAD

**Issue by dataopt** [rstudio/bookdown#568]
http://wave.webaim.org/report#/https://bookdown.org/yihui/bookdown/ shows that bookdown colour schemes have contrast issues for the default and sepia themes:

![bookdown_constrast_errors](https://user-images.githubusercontent.com/28999218/38778946-2b9416b6-408f-11e8-9a15-298930d96cc4.png)

![bookdown_sepia_constrast_issues](https://user-images.githubusercontent.com/28999218/38778949-35a9536e-408f-11e8-8acf-a2963a210dbf.png)

These issues lead to books generated by bookdown not being able to meet the WCAG 2.0 requirements.

To correct these issues requires changes to the gitbook CSS.

**Comment by kcudding** [comment]
Yup, just ran into this during review from accessibility office. Looking for a custom .css solution right now, but meeting the standard should be the default.
